config/initializers/date_formats.rb

```diff
@@ -1 +1,2 @@
 Date::DATE_FORMATS[:nomis] = '%Y-%m-%dT%H:%M:%S'
+Time::DATE_FORMATS[:nomis] = '%Y-%m-%dT%H:%M:%S'
```

Comment:
could you add a comment so we won't forget Nomis format is local London time:
`# NB: Nomis is in a London time zone (i.e. GMT or BST depending on time of year)`

Reply:
Sure!